DQM/SiStripMonitorClient/interface/SiStripDcsInfo.h

```diff
@@ -86,6 +86,7 @@ class SiStripDcsInfo: public edm::EDAnalyzer {
     MonitorElement* DcsFractionME;
     int TotalDetectors;
     std::vector<uint32_t> FaultyDetectors;
+    std::map<uint32_t,uint32_t> NLumiDetectorIsFaulty;
```

Comment:
std::unordered_map should be used instead of std::map. Otherwise, perhaps 16 bits is enough for the second argument (65000 events before overflowing)?